components/operations-controller/Makefile

```diff
@@ -1,6 +1,6 @@
-APP_NAME = compass-operations-controller
+APP_NAME = compass-operations-controller-manager
 APP_PATH = components/operations-controller
-BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20210125-6234473e
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/pr/buildpack-golang-kubebuilder2:PR-3299
```

Comment:
https://github.com/kyma-project/test-infra/pull/3299 has been merged. Use the newly built image as a result of it, instead of the one tagged with the PR.